.github/workflows/documentation.yml

```
@@ -0,0 +1,36 @@
+name: Documentation
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+  
+jobs:
+  make-documentation:
+    runs-on: ubuntu-latest
+    steps:
+    
+      - name: Check out sopt
+        uses: actions/checkout@v2
+    
+      - name: Install dependencies
+        run: sudo apt install libfftw3-dev libtiff5-dev libboost-all-dev libeigen3-dev libyaml-cpp-dev libcfitsio-dev casacore-dev doxygen graphviz
```

It runs successfully for me with this subset of dependencies
```suggestion
        run: sudo apt install libtiff5-dev openmpi-bin libopenmpi-dev libeigen3-dev libyaml-cpp-dev doxygen graphviz
```